python/kfserving/kfserving/api/kf_serving_client.py

```diff
@@ -202,6 +202,27 @@ def replace(self, name, kfservice, namespace=None, watch=False, timeout_seconds=
             return outputs
 
 
+    def rollout_canary(self, name, percent=100, namespace=None, # pylint:disable=too-many-arguments,inconsistent-return-statements
```

Comment:
Hmm.. wonder if percent should be a required field? As 100 percent seems like an extreme side effect if all the user wanted todo was 0 and forgot to specify percent by mistake?

Reply:
Make sense, that's better to be required. Thanks @rakelkar